src/Particular.ThroughputCollector.Persistence.RavenDb/Particular.ThroughputCollector.Persistence.RavenDb.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="ByteSize" />
+    <!--<PackageReference Include="NServiceBus.CustomChecks" />-->
+    <PackageReference Include="NServiceBus" />
```

Comment:
I don't think that we should be referencing nservicebus - I've been using the microsoft.extensions.logging everywhere.